MANIFEST.in

```diff
@@ -1,4 +1,3 @@
-include cylc/uiserver/logging_config.json
```

Comment:
Should the actual `logging_config.json` file not be deleted also?

Reply:
Done.